### PR TITLE
website/docs: added a Docs banner to announce new docs structure

### PR DESCRIPTION
**Pull request by tanberry**
This PR adds a banner to our Docs website, announcing the upcoming change to the structure of the docs, and providing a link to the deploy preview.

-   [ ] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docusaurus.config.ts [hunk 1]
```diff
@@ -17,6 +17,14 @@ module.exports = async function (): Promise<Config> {
         organizationName: "Authentik Security Inc.",
         projectName: "authentik",
         themeConfig: {
+            announcementBar: {
+                id: "support_us",
+                content:
+                    'Change is hard. We hate it too when a familiar site or application gets re-arranged. But we did it anyway. Please let us <a target="_blank" rel="noopener noreferrer" href="https://docs.goauthentik.io/docs/">know what you think of the new Docs structure!</a>',
```
Comment (kensternberg-authentik) [inline website/docusaurus.config.ts:23]: Oh! I was thinking "But we did it anyway, and we think the new layout is a lot smarter and easier to navigate.  Let us know what you think."  But the banner itself is unremarkable, so approved.

**Review by kensternberg-authentik — APPROVED** [review]

**Review by rissson — APPROVED** [review]

**Comment by tanberry** [comment]
cherry-pick version-2024.8

**Pull request merged**